lib/package.rb

```diff
@@ -164,13 +164,19 @@ def self.system(*args)
       { exception: true }
     end
 
-    # add -j arg to build commands
+    # Add -j arg to build commands.
     cmd_args.sub(/\b(?<=make)(?=\b)/, " -j#{CREW_NPROC}") unless cmd_args =~ /-j\s*\d+/
+    # Escape special bash characters.
+    cmd_args.gsub!('"','\\\\"')
+    cmd_args.gsub!('$','\\\\$')
+    cmd_args.gsub!('`','\\\\`')
 
     begin
       # use bash instead of /bin/sh
       @system_args = ''
       [@system_args, "bash -e -c \"#{cmd_args}\""].reduce(&:concat)
+      # Uncomment following line for debugging.
+      # puts @system_args.orange
```

Comment:
Rather than uncommenting for debugging, why not add a `-d` switch option?  Should be just a few additional lines of code needed in crew.

Reply:
Sorry, `-d` is for disabling color.  Maybe `-D`?

Reply:
I left that just for debugging optimization. If I recall correctly the standard debut flags don't work in `package.rb`...

We can take it out after we are sure this all works well.